cypress/plugins/index.js

```diff
@@ -0,0 +1,10 @@
+const wp = require('@cypress/webpack-preprocessor');
```

Comment:
Do we need to use a scoped (`@cypress`) require here instead of just `cypress`?

Reply:
`const wp = require('cypress/webpack-preprocessor');` doesn't seem to work, if that's what you meant?